.github/workflows/Deploy.yaml

```diff
@@ -1,5 +1,7 @@
 name: Deploy
 
+run-name: "Deploy from [${{ github.ref_name }}] to [${{ inputs.branch }}]"
```

Comment:
doesn't that need to be below the on: section?

Reply:
No, it works fine as it is.